### PR TITLE
Alg2 - exponential models

### PR DESCRIPTION
**Pull request by schanzer**
I know that there's a LOT of unpicked fruit here as far as practicing the computational skills required to work with exponents. With the timeline so short, I just don't think I'm going to be able to get to it by Thursday. So for now the main question is: does this lesson hang together? Is it the right mix of Desmos and Pyret? Is it the right mix of engagement and rigor?

### CODE REVIEW

File: lessons/Algebra2/exploring-linear-models/langs/en-us/pages/model-college-v-income.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+@n Change the definition of `f` so that it draws a horizontal line. What is the @math{R^2} value? @fitb{}{}
+
+@n What do you think @math{R^2} tells us? @fitb{}{@ifsoln{It measures how well the model fits the data}}
+
+== Build a Model through Trial {amp} Error
+
+@n Try using Massachusetts (row 21) and Nevada (row 28) instead of Alabama and Alaska. What @math{R^2} do you get? @fitb{}{@ifsoln{2.84}}
+
+@n What is the @math{R^2} for **the best model you can come up with**? @fitb{5em}{} Write the model below.
+
```
Comment (retabak) [inline lessons/Algebra2/exploring-linear-models/langs/en-us/pages/model-college-v-income.adoc:53]: Just occurred to me that the above question might be confusing. Are they just messing around and changing the slope and y-intercept however they'd like? Or should they try different pairs of states? (When I first did this worksheet, I thought it was the former, but the inclusion of Q9 (above, about Massachusetts and Nevada) has me wondering if I was wrong?

File: lessons/Algebra2/exploring-linear-models/langs/en-us/pages/graphing-models.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		linear-gradient(rgba(150, 150, 150, .3) 1px, transparent 1px),
+		linear-gradient(90deg, rgba(150, 150, 150, .3) 1px, transparent 1px);
+	background-size: 100px 100px, 100px 100px, 20px 20px, 20px 20px;
+	background-position: 56px 23px;
+}
+
+td:last-child { align-items: center; justify-items: center; }
+</style>
+++++
+
```
Comment (retabak) [inline lessons/Algebra2/exploring-linear-models/langs/en-us/pages/graphing-models.adoc:23]: This page looks MUCH better than the last time I saw it, both in terms of layout/aesthetics and its purposefulness.

File: lessons/Algebra2/exploring-linear-models/langs/en-us/pages/graphing-models.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+Choose three linear models from @opt-printable-exercise{model-your-own}: your first guess, a closer guess, and then your *best* guess. For graph each model in the squares below, being sure to draw your axes. Then label the slope, y-intercept, and @math{R^2} value of each model!
+
```
Comment (retabak) [inline lessons/Algebra2/exploring-linear-models/langs/en-us/pages/graphing-models.adoc:25]: Oops - last sentence of para above needs to be fixed; doesn't make sense right now.

File: lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+- Using `lr-plot` in Pyret, come up with the best possible linear model, and write your answer at the bottom of @printable-exercise{exploring-covid.adoc}.
+- Class discussion: Are linear models a good fit for this data? Why or why not?
+
+Linear models capture _straight-line relationships_, where one quantity varies proportionally based on another. In linear models, we expect the response variable to grow by equal amounts over equal intervals in the explanatory variable.
+
+@right{@image{images/MA-covid.png}} If we make the line go from the start to the peak, almost all of the points bulge out below out line of best fit. If we make the line fit the bulge, all the points fall above it! We always have either too many points _below_ the line in the middle or _above_ the line at the end. **It's growing too fast to be fit with a linear model that grows at a constant rate!**
+
+[.lesson-instruction]
+- Take a few minutes to explore @online-exercise{https://www.desmos.com/calculator/73y2med4yj, MA Covid Spread - Quadratic}.
+- Complete the *second* section on @printable-exercise{graphing-models.adoc}.
```
Comment (retabak) [inline lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc:112]: Too much going on in this lesson-instruction. I'd inlcude the first two bullets, then some discussion prompts. Feels overwhelming to have two different platforms in such a quick succession. 

There could be some text to offer teachers guidance on how to help kids make connections between desmos and Pyret and why we've chosen to include desmos before you jump back to Pyret.

(Come to think of it, teachers might like an explanation for the inclusion of Desmos when it first appears. Could disucss how differnet tools have different affordances, blah blah.)

File: lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+- Money in a savings account grows by a certain percentage each year, meaning every year there's that much more money to grow
+
+[.lesson-point]
+Linear functions grow by equal *amounts* over equal intervals (adding _n_ each time). Exponential functions grow by equal *factors* over equal intervals (multiplying by _n_ each time).
+
+[.lesson-instruction]
+Can you think of other real-world examples where a relationship is likely exponential?
+
+Exponential models have the form @math{y = ab^x + k}
+
```
Comment (retabak) [inline lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc:137]: I would add, "As you discovered while exploring desmos, a is the initial value, b is a growth factor, and k is the vertical shift."

(You probably want to elaborate more on these.)

File: lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+=== Investigate
+
```
Comment (retabak) [inline lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc:139]: If you were going to do 3 sections to this lesson, the first would be Simpson's Paradox, the 2nd would be the first two desmos activities, and the third would start here, with the final desmos activity + the fit-model worksheet that doesn't exist.

**Review by retabak — COMMENTED** [review]
OVERALL: I'm pretty sure this lesson needs 3 sections, not 2. 
(Or you could do a whole lesson on filtering + Simpson's Paradox + a 2-section lesson on exponential models... but that would require more time, which I know you don't have.)

File: lessons/Algebra2/exploring-exponential-models/langs/en-us/pages/graphing-models.adoc [hunk 7]
```diff
@@ -0,0 +1,102 @@
+= Graphing Exponential Models
+++++
```
[the remark this answers is not in the document]
Comment (retabak) [reply lessons/Algebra2/exploring-exponential-models/langs/en-us/pages/graphing-models.adoc:2]: Coming back to add that I think this is crucial!

File: lessons/Algebra2/exploring-linear-models/langs/en-us/pages/model-college-v-income.adoc (hunk 1 above)
Comment (schanzer) [reply lessons/Algebra2/exploring-linear-models/langs/en-us/pages/model-college-v-income.adoc:53]: Clarified the instructions. :)

File: lessons/Algebra2/exploring-linear-models/langs/en-us/pages/other-forms-linear-models.adoc [hunk 8]
```diff
@@ -0,0 +1,52 @@
+= Other Forms of Linear Models
+++++
+<style>
+.studentAnswerMedium { min-width: 8em !important; }
+body.workbookpage .fitbruby { padding-top: 1.1rem; }
+body:not(.LessonPlan, .narrativepage) p { line-height: 1.2; }
+.center { padding: 0; }
+</style>
+++++
+
```
[the remark this answers is not in the document]
Comment (schanzer) [reply lessons/Algebra2/exploring-linear-models/langs/en-us/pages/other-forms-linear-models.adoc:10]: Good call! Done

File: lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/graphing-models.adoc [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+== Horizontal Translation @math{h}
+@n Set @math{a} back to 1. Change the value of @math{h} to -20, 0, and 10, graphing each parabola in the squares below and labeling the vertex and roots.
+
+
+[.FillVerticalSpace, cols="1,1,1", frame="none"]
+|===
+| @math{h=-20}	| @math{h=0}	| @math{h=10}
+|===
+
```
[the remark this answers is not in the document]
Comment (schanzer) [reply lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/graphing-models.adoc:48]: OK for me to limit to -5 and 5?

File: lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/graphing-models.adoc [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+@n What does @math{h} tell us about a parabola? @fitb{}{@ifsoln{How far the parabola is shifted left (negative h) or right (positive h)}} +
+
+
+== Vertical Translation @math{k}
+@n Set @math{h} back to 0. Change the value of @math{k} to -15, 0, and 15, graphing each parabola in the squares below and labeling the vertex and roots.
+
```
[the remark this answers is not in the document]
Comment (schanzer) [reply lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/graphing-models.adoc:54]: OK for me to limit to -5 and 5? Resolve if yes.

File: lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/graphing-models.adoc [hunk 11]
```diff
@@ -0,0 +1,62 @@
+= Graphing Quadratic Models
+++++
```
[the remark this answers is not in the document]
Comment (schanzer) [reply lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/graphing-models.adoc:2]: Good call. I believe the changes I've made address this.

File: lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc [hunk 12]
```diff
@@ -0,0 +1,163 @@
...
+=== Launch
+
+[.lesson-instruction]
+- Open the @online-exercise{https://code.pyret.org/editor#share=1ueyf5-0ByVYmeJKQ03i9msb9u1C5y50o, Covid Spread Starter File}
+- From the File menu, select "Save a Copy", and click "Run"
+- Working in pairs or small groups, complete questions 1-8 of @printable-exercise{exploring-covid.adoc}
+
+@center{@image{images/multiple-models.png}}
+
```
[the remark this answers is not in the document]
Comment (schanzer) [reply lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc:50]: GREAT idea!

File: lessons/Algebra2/exploring-exponential-models/langs/en-us/pages/graphing-models.adoc [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+	background-position: -3px 74px;
+	min-height: 1.5in;
+	max-width: 2in;
+}
+
+td .fitb { padding-top: 0 !important; }
+td .MathJax { min-width: 30px !important; display: inline-block; }
+</style>
+++++
+
```
[the remark this answers is not in the document]
Comment (schanzer) [reply lessons/Algebra2/exploring-exponential-models/langs/en-us/pages/graphing-models.adoc:29]: OMG How does one hide the table???

[the remark this answers is not in the document]
Comment (retabak) [reply lessons/Algebra2/exploring-exponential-models/langs/en-us/pages/graphing-models.adoc:29]: This isn't exactly hiding it completely but I think it is better than what we've got currently: https://www.youtube.com/watch?v=3m4Bma2LGuU

**Pull request closed without merging**